setup.bash

```
@@ -4,11 +4,23 @@ echo "SETUP SCRIPT FOR CODEFORT"
+for arg in "$@"; do
+  if [[ "$arg" == "--assume-yes" ]] || [[ "$arg" == "-y" ]]; then
+    assume_yes=1
+    break
+  fi
+done
+
+# Skip confirmation if --assume-yes or -y flag is present
+if [[ "$assume_yes" -eq 0 ]]; then
+  read -r -p "Are you sure you want to proceed? [y/N] " -n 1
+  echo
+  if [[ !("$REPLY" =~ ^[Yy]$) ]]; then
```

Shellcheck (0.10.0)

[error] You are missing a required space here.

(SC1035)